scss/_orange-navbar.scss

```diff
@@ -23,6 +23,10 @@
       padding-top: add(var(--#{$prefix}navbar-nav-icon-padding-y), .3125rem);
     }
   }
+
+  &.navbar-nav .nav-link:not(.nav-icon) {
```

Comment:
Are you sure about the `:not(.nav-icon)`? Could you explain me the tests that you've done to decide to exclude it?

I've tried something like this and seemed better with the `margin-right: 0` rule:

```diff
diff --git a/site/layouts/shortcodes/orange-global-headers.html b/site/layouts/shortcodes/orange-global-headers.html
index 4bf1c93da..ce8bba6ef 100644
--- a/site/layouts/shortcodes/orange-global-headers.html
+++ b/site/layouts/shortcodes/orange-global-headers.html
@@ -83,6 +83,16 @@
         <li class="nav-item"><a class="nav-link active" href="#" aria-label="EN English version" aria-current="true">EN</a></li>
         <li class="nav-item"><a class="nav-link" href="#" aria-label="FR Version française" lang="fr" hreflang="fr">FR</a></li>
         <li class="nav-item"><a class="nav-link" href="#" aria-label="ES Versión en español" lang="es" hreflang="es">ES</a></li>
+        <li class="nav-item{{ cond $responsive_logo " d-none" "" }}">
+          <a href="#" class="nav-link nav-icon">
+            <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
+              <use xlink:href="/docs/{{ $.Site.Params.docs_version }}/assets/img/boosted-sprite.svg#buy" />
+            </svg>
+          </a>
+        </li>
+        <li class="nav-item"><a class="nav-link active" href="#" aria-label="EN English version" aria-current="true">EN</a></li>
+        <li class="nav-item"><a class="nav-link" href="#" aria-label="FR Version française" lang="fr" hreflang="fr">FR</a></li>
+        <li class="nav-item"><a class="nav-link" href="#" aria-label="ES Versión en español" lang="es" hreflang="es">ES</a></li>
       </ul>
 {{- end }}
     </div>
```